runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -73,7 +73,7 @@ default boolean lockWindowSize()
 	)
 	default boolean enablePlugins()
 	{
-		return true;
+		return false;
```

Comment:
Not needed, just make hide externals by default in plugin sorter.

Reply:
> Not needed, just make hide externals by default in plugin sorter.

that hides everything, not just external plugins. this way it keeps JUST externals disabled by default.